manual/sphinx/user_docs/installing.rst

```diff
@@ -286,8 +286,16 @@ CMake
 -----
 
 There is now (experimental) support for `CMake <https://cmake.org/>`_. You will need CMake >
-3.9. CMake supports out-of-source builds by default, which are A Good
-Idea. Basic configuration with CMake looks like::
+3.9. Note that it is possible to get the latest version of CMake using ``pip``::
+
+  $ pip install --user cmake
```

Comment:
maybe add `-U` to install the most recent version?

Reply:
I prefer long-form so it's more obvious to the user why the argument's there
```suggestion
  $ pip install --user --upgrade cmake
```